js/main.js

```diff
@@ -19,4 +19,91 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(entry);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+
+  $uList.prepend(renderEntry(entry));
+  viewSwap(event);
```

Comment:
You shouldn't be passing `event` here, you want to pass a string representing the view you want.